src/cvsim/fit_curve.py

```diff
@@ -315,26 +353,297 @@ def fetch(param: str) -> float:
+    disk_radius : float
+        Radius of disk macro-electrode (mm).
+    temperature : float
+        Temperature (K).
+        Default is 298.0 K (24.85C).
+    reduction_potential : float | None
+        Reduction potential of the one-electron transfer process (V vs. reference).
+        If known, can be fixed value, otherwise defaults to None.
+    diffusion_reactant : float | None
+        Diffusion coefficient of reactant (cm^2/s).
+        If known, can be fixed value, otherwise defaults to None.
+    diffusion_product : float | None
+        Diffusion coefficient of product (cm^2/s).
+        If known, can be fixed value, otherwise defaults to None.
+    alpha : float | None
+        Charge transfer coefficient (no units).
+        If known, can be fixed value, otherwise defaults to None.
+    k_0 : float | None
+        Standard electrochemical rate constant (cm/s).
+        If known, can be fixed value, otherwise defaults to None.
+
+    """
+
+    def __init__(
+            self,
+            voltage_to_fit: list[float] | np.ndarray,
+            current_to_fit: list[float] | np.ndarray,
+            scan_rate: float,
+            c_bulk: float,
+            step_size: float,
+            disk_radius: float,
+            temperature: float = 298.0,
+            reduction_potential: float | None = None,
+            diffusion_reactant: float | None = None,
+            diffusion_product: float | None = None,
+            alpha: float | None = None,
+            k_0: float | None = None,
+    ) -> None:
+        super().__init__(
+            voltage_to_fit,
+            current_to_fit,
+            scan_rate,
+            c_bulk,
+            step_size,
+            disk_radius,
+            temperature,
+            reduction_potential,
+            diffusion_reactant,
+            diffusion_product,
+        )
+
+        self._ensure_open_unit_interval_or_none('alpha', alpha)
+        self._ensure_positive_or_none('k_0', k_0)
+
+        self.alpha = alpha
+        self.k_0 = k_0
+
+        # Contains only variables with a user-specified fixed value
+        self.fixed_vars = self._non_none_dict({
+            'reduction_potential': reduction_potential,
+            'diffusion_reactant': diffusion_reactant,
+            'diffusion_product': diffusion_product,
+            'alpha': alpha,
+            'k_0': k_0,
+        })
+
+        # Values are [initial guess, lower bound, upper bound]
+        self.default_vars = {
+            'reduction_potential': [
+                round((self.voltage_to_fit[np.argmax(self.current_to_fit)]
+                       + self.voltage_to_fit[np.argmin(self.current_to_fit)]) / 2, 3),
+                round(min(self.start_voltage_mv, self.reverse_voltage_mv) / 1000, 3),
+                round(max(self.start_voltage_mv, self.reverse_voltage_mv) / 1000, 3),
+            ],
+            'diffusion_reactant': [1e-6, 5e-8, 1e-4],
+            'diffusion_product': [1e-6, 5e-8, 1e-4],
```

Comment:
Similar comment here, we should avoid respecifying these values (unless we anticipate a case where we would want different default values under different schemes for the same var).

Reply:
done